src/utils/saml.ts

```diff
@@ -554,7 +554,11 @@ const mainAttributeValidation = (
     .chain(IssueInstant => utcStringToDate(IssueInstant, "IssueInstant"))
     .chain(
       fromPredicate(
-        _ => !hasStrictValidation || _.getTime() < Date.now(),
+        _ =>
+          _.getTime() <
```

Comment:
can we move this computation at caller level (at the same level of hasStrictValidation) so we can reuse it seamlessy between inner methods ? ie.

```
const acceptedClockSkewMs = (samlConfig.acceptedClockSkewMs || 0) === -1
            ? Infinity
            : Date.now() + samlConfig.acceptedClockSkewMs);
```

Reply:
In two cases the value is `Infinity` (NotBefore and IssueInstant) and on another case is `-Infinity` (NotOnOrAfter). How we can handle these scenarios?

Reply:
sorry I didn't see this, it's ok as is then